lib/constants.go

```diff
@@ -1110,7 +1110,7 @@ var DeSoMainnetParams = DeSoParams{
 	MinChainWorkHex: "000000000000000000000000000000000000000000000000006314f9a85a949b",
 
 	MaxTipAgePoW: 24 * time.Hour,
-	MaxTipAgePoS: time.Hour,
+	MaxTipAgePoS: 24 * time.Hour,
```

Comment:
Ok so I have a question: If we're syncing from scratch, do we use MaxTipAgePoS to mark when we start checking block signatures? Because that would make sync take a lot longer unnecessarily. If this is NOT the case, then I like 24h as the time. If it IS the case, then maybe just do 3h or something more conservative until we can break that signature checking thing into its own flag...

Reply:
I'll lower to 3 hours - this does make us start checking signatures sooner.